pkgs/by-name/co/code-nautilus/package.nix

```
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nautilus-python,
+  python3,
+}:
+stdenv.mkDerivation rec {
+
+  pname = "code-nautilus";
+  version = "0-unstable-2024-09-11";
+
+  src = fetchFromGitHub {
+    owner = "harry-cpp";
+    repo = "code-nautilus";
+    rev = "8ea0ce78f3f1f7a6af5f9e9cf93fc3e70015f61e";
+    sha256 = "u10laS3BwUVCJYlQ6WivU7o/sFFv6cTeV+uxBEdD7oA=";
+  };
+
```

```suggestion
  postPatch = ''
    mv code-nautilus.py code-nautilus-raw.py
    echo "#!/usr/bin/env python3" > code-nautilus.py
    cat code-nautilus-raw.py >> code-nautilus.py
  '';
```
Otherwise the script will not pick up our Python.